src/components/pages/devices/flyouts/deviceDetails/deviceDetails.js

```diff
@@ -305,16 +310,16 @@ export class DeviceDetails extends Component {
                       </Row>
                       {
                         device.connected && [
-                        <Row key="diag-row-time">
-                          <Cell className="col-3">{t('devices.flyouts.details.diagnostics.lastMessage')}</Cell>
-                          <Cell className="col-7">{(lastMessage || {}).time}</Cell>
-                        </Row>,
-                        <Row key="diag-row-msg">
-                          <Cell className="col-3">{t('devices.flyouts.details.diagnostics.message')}</Cell>
-                          <Cell className="col-7">
-                            <Btn className="raw-message-button" onClick={() => this.setState({ showRawMessage: !this.state.showRawMessage })}>{t('devices.flyouts.details.diagnostics.showMessage')}</Btn>
-                          </Cell>
-                        </Row>
+                          <Row key="diag-row-time">
+                            <Cell className="col-3">{t('devices.flyouts.details.diagnostics.lastMessage')}</Cell>
+                            <Cell className="col-7">{(lastMessage || {}).time}</Cell>
+                          </Row>,
+                          <Row key="diag-row-msg">
+                            <Cell className="col-3">{t('devices.flyouts.details.diagnostics.message')}</Cell>
+                            <Cell className="col-7">
+                              <Btn className="raw-message-button" onClick={() => this.setState({ showRawMessage: !this.state.showRawMessage })}>{t('devices.flyouts.details.diagnostics.showMessage')}</Btn>
```

Comment:
The onClick should be function from the class instance instead of making a new function in the JSX.